content/blog/2018/istio-authorization/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Micro-Segmentation with Istio Authorization
+description: Describe Istio authorization characterics and how it can be used for different use cases.
+publishdate: 2018-07-20
+subtitle:
+attribution: Limin Wang
+weight: 87
+keywords: [authorization, Role Based Access Control, security]
+---
+
+[Istio's authorization feature](/docs/concepts/security/#authorization), also known as Istio Role Based Access Control,
```

Comment:
I would start with defining what Micro-segmentation is.
Micro-segmentation is a security technique that enables fine-grained security policies to be assigned to data center applications, down to the workload level. source: https://www.sdxcentral.com/sdn/network-virtualization/definitions/how-does-micro-segmentation-help-security-explanation/